locales/en/messages.json

```diff
@@ -2311,15 +2474,27 @@
         "message": "deep pink"
     },
     "controlAxisRoll": {
-        "message": "Roll [A]"
+        "message": "Roll"
```

Comment:
Not sure if this should be done. `AETR` seem to be the commonly used designators for the axes in fixed wing flying.

Reply:
I didn't know, I thought these were shortcuts for the MSP virtual remote... I will change it.

Reply:
They stand for [A]ileron, [E]levator, [T]rottle, [R]udder, which are well defined in fixed wing aircraft.